natlas-server/package.json

```diff
@@ -1,4 +1,10 @@
 {
+	"name": "natlas-web",
+	"description": "Natlas web app logic",
+	"license": "Apache-2.0",
+	"homepage": "https://github.com/natlas",
+	"bugs": "https://github.com/natlas/natlas/issues",
+	"repository": "github:natlas/natlas",
```

Comment:
Should this be `github.com:natlas/natlas`?

Reply:
According to yarn docs this format is correct, short of including the whole hyperlink.

 https://classic.yarnpkg.com/en/docs/package-json/#toc-repository